### PR TITLE
Export Log: pDialog Child Log Table

### PR DESCRIPTION
**Pull request by anishfyle**

### CODE REVIEW

File: src/app/core/models/db/accounting-export.model.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ export interface ExportableAccountingExport {
 }
 
 export interface AccountingExportList {
+    index?: number;
```
Comment (ashwin1111) [inline src/app/core/models/db/accounting-export.model.ts:19]: remove index

File: src/app/shared/components/export-log/export-log-table/export-log-table.component.ts [hunk 2]
```diff
@@ -20,6 +21,10 @@ export class ExportLogTableComponent implements OnInit {
 
   @Input() isDashboardFailed: boolean;
 
+  clickedExportLogIndex: number = 0;
```
Comment (ashwin1111) [inline src/app/shared/components/export-log/export-log-table/export-log-table.component.ts:24]: not needed

File: src/app/shared/components/export-log/export-log-table/export-log-table.component.ts [hunk 3]
```diff
@@ -28,6 +33,12 @@ export class ExportLogTableComponent implements OnInit {
     private windowService: WindowService
   ) { }
 
+  displayChildTable(index: number) {
```
Comment (ashwin1111) [inline src/app/shared/components/export-log/export-log-table/export-log-table.component.ts:36]: change approach and assign back review

**Comment by anishfyle** [comment]
@coderabbitai review

File: src/app/shared/components/export-log/export-log-dialog/export-log-child-table-dialog.component.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+:host ::ng-deep .p-dialog {
+    @apply tw-max-h-full;
+    margin: 0rem !important;
```
Comment (ashwin1111) [inline src/app/shared/components/export-log/export-log-dialog/export-log-child-table-dialog.component.scss:3]: tailwind

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**